CHANGELOG.asciidoc

```diff
@@ -23,6 +23,8 @@ https://github.com/elastic/beats/compare/v6.5.0...6.5[Check the HEAD diff]
 
 *Metricbeat*
 
+- Fix issue preventing diskio metrics collection for idle disks. {issue}9124[9124] {pull}9125[9125]
```

Comment:
This sounds more like a bug fix then a breaking change?

Reply:
And cherry-picked the fix in the changelog here to avoid one more PR.